src/ansys/dpf/core/field_base.py

```diff
@@ -494,6 +494,12 @@ def _get_data(self, np_array=True):
     def _set_data(self, data):
         pass
 
+    def __enter__(self):
```

Comment:
@ansys-akarcher is that needed to do with statement?

Reply:
@cbellot000 
__enter__ and __exit__ are indeed required as now as_local_field() may be returning either a Field Object or a LocalField (which was defining __enter__ and __exit__)